plugins/renepay/routetracker.h

```diff
@@ -7,16 +7,22 @@
 #include <plugins/renepay/route.h>
 
 struct routetracker{
-	struct payment *payment;
+	/* Routes that we compute and are kept here before sending them. */
+	struct route **computed_routes;
+
+	/* Routes that we sendpay and are still waiting for rpc returning
+	 * success. */
 	struct route_map *sent_routes;
-	struct route_map *pending_routes;
+
+	/* Routes that have concluded (either SENDPAY_FAILED or
+	 * SENDPAY_COMPLETE). */
 	struct route **finalized_routes;
```

Comment:
Note to myself: make sure all this extra data gets cleaned after the payment session concludes.